.github/workflows/container_scanning.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - uses: anchore/scan-action@master
       with:
         dockerfile-path: "Dockerfile"
-        image-reference: "openjdk:8-jdk-alpine"
+        image-reference: "openjdk:oraclelinux7"
```

Comment:
@nleach999 why the image was changed ?

Reply:
Alpine Linux lacks support for packages such as the Perforce client.  Most enterprise tools (assuming someone wants to use some other SCM or build tool in a child image) will likely want a RedHat derivative.  Oracle Linux is a fairly common choice for enterprise Linux.

Reply:
@nleach999 good to know, thanks ;)